### DISCUSSION TITLE
Is there any possibility to have a record option in the image format.

### DISCUSSION THREAD

**Discussion by BharadwajManne** [bluenviron/mediamtx#2890] [Questions]
### Question

Is there any possibility of having a record option in the image format like jpeg or PNG

**Comment by dragulesq** [comment]
In the `runOnReady:` section you could fave ffmpeg capture the local endpoint and spit out images, something like:
```
runOnReady: >
  ffmpeg  -i rtsp://localhost:8554/mypath ./images/%05d.jpg
```
^ this assumes a rtsp source/path

**Reply by aler9** [reply]
Exactly.

Regarding making this feature native, we're not gonna do it because it would require decoding streams, and that's something this server avoids to do.

[a comment by an automated account is not shown]
